frontend/package.json

```diff
@@ -3,11 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/styles": "^4.10.0",
```

Comment:
We should have @material-ui/icons as a dependency right? Building it using the docker image complained about that and it seems like StudentAssignments.js and NavBar.js require it

Reply:
Since it is just that one problem, I will just merge this and resolve the dependencies for all the pull requests at the same time